201-virtual-wan-with-all-gateways/README.md

```diff
@@ -0,0 +1,81 @@
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/201-virtual-wan-with-all-gateways/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/201-virtual-wan-with-all-gateways/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/201-virtual-wan-with-all-gateways/CredScanResult.svg)
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-virtual-wan-with-all-gateways%2Fazuredeploy.json)
+
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-virtual-wan-with-all-gateways%2Fazuredeploy.json)
+
+## Solution Overview
+
+This template creates a fully functional Azure Virtual WAN (vWAN) environment with the following resources:
+
+- Two distinct hubs in different regions
+- Four Azure Virtual Networks (VNET)
+- Two VNET connections for each vWAN hub
+- One Point-to-Site (P2S) VPN gateway in each hub
+- One Site-to-Site (S2S) VPN gateway in each hub
+- One Express Route gateway in each hub
+
+## Architecture
+
+vWAN resource deployed is of type "Standard" with default full mesh connectivity.
+The scenario implemented is exactly the one referenced in the Azure Virtual WAN documentation article below:
+
+[Azure vWAN Routing Scenario: any-to-Any](https://docs.microsoft.com/azure/virtual-wan/scenario-any-to-any)
+
+:::image type="content" source="images/azurevirtualwanarchitecturefigure1.jpg" alt-text="vWAN Architecture":::
+
+List of input parameters has been kept at the very minimum, for each one a default value has been provided to expedite sample creation.
+IP addressing scheme can be changed modifying the variables inside the template, default values have been provided based on the architecture diagram above.
+
+> [!NOTE]
+> This template will create all the vWAN resources listed above, but will not create the customer side resources required for hybrid connectivity. After template deployment will be completed, user will need to create P2S VPN clients, VPN branches (Local Sites) and connect Express Route circuits.
+
+> [!WARNING]
+> Default values for parameters ***Hub1_PublicCertificateDataForP2S*** and ***Hub2_PublicCertificateDataForP2S***  (Point-to-Site (P2S) configuration) have been provided only as a sample to complete quickly the vWAN deployment procedure. After the deployment is completed, you should generate your own certificates and change each Point-to-Site (P2S) Gateway configuration to use your own.
```

Comment:
some of the defaultValues need to be removed, so this section will need to be updated